x/ccv/provider/keeper/relay.go

```diff
@@ -273,6 +290,12 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 		slashFraction sdk.Dec
 	)
 
+	// Obtain infraction height or panic
+	infractionHeight, found := k.getMappedInfractionHeight(ctx, chainID, data.ValsetUpdateId)
+	if !found {
+		panic("infraction height not found. But was found during slash packet validation")
```

Comment:
This will panic if someone forgets to clean up the slash packet queues

Reply:
now just a log 👍